src/main/java/tb/common/block/BlockTBPlant.java

```diff
@@ -140,6 +143,43 @@ public void func_149853_b(World w, Random r, int x, int y, int z) {
             3);
     }
 
+    @Override
+    public boolean onBlockActivated(World aWorld, int aX, int aY, int aZ, EntityPlayer aPlayer, int aSide, float pX,
+        float pY, float pZ) {
+        int aMeta = aWorld.getBlockMetadata(aX, aY, aZ);
+        // check for Growth Stage
+        if (aMeta >= growthStages - 1) {
+            // eval fortune on rightclick
+            int fortune = 0;
+            ItemStack heldItem = aPlayer.getHeldItem();
+            if (heldItem != null) {
+                if (heldItem.getEnchantmentTagList() != null) {
+                    for (int i = 0; i < heldItem.getEnchantmentTagList()
+                        .tagCount(); ++i) {
+                        if (heldItem.getEnchantmentTagList()
+                            .getCompoundTagAt(i)
+                            .getInteger("id") == Enchantment.fortune.effectId) {
+                            fortune = heldItem.getEnchantmentTagList()
+                                .getCompoundTagAt(i)
+                                .getInteger("lvl");
+
+                            // Damage fortune item on crop break?
+                            // heldItem.setItemDamage(heldItem.getItemDamage() + 1);
+                            break;
+                        }
+                    }
+                }
```

Comment:
This can easily be done with `EnchantmentHelper`.

Reply:
Working on it.